eis_qgis_plugin/wizard/eda/plot_utils.py

```diff
@@ -99,3 +101,102 @@ def generate_color_mapping(
 
 def opacity_to_alpha(opacity: float):
     return opacity * 256
+
+
+class RasterTable(QTableWidget):
```

Comment:
Perhaps "PlotDataTable" would be more fitting name since all the tables in the file are raster tables